src/cupbearer/utils/__init__.py

```diff
@@ -36,6 +39,7 @@ def validate_and_convert_leaf(leaf):
     if isinstance(leaf, type):
         return TYPE_PREFIX + leaf.__module__ + "." + leaf.__name__
 
+    # TODO (erik): is any of this still necessary? torch.save should handle this I think
```

Comment:
Good idea to add names to TODOs. I should start doing that as well.

Reply:
Yeah, I'm not doing it consistently yet, but should probably get used to it as more people start working on the codebase